apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -84,6 +88,7 @@ private ApolloClient(Builder builder) {
     this.defaultHttpCacheControl = builder.defaultHttpCacheControl;
     this.defaultCacheControl = builder.defaultCacheControl;
     this.logger = builder.apolloLogger;
+    this.applicationInterceptors = builder.applicationInterceptors;
```

Comment:
Never mind we don't set in builder the list of interceptors but add them one by one